gallery/src/pageConfig.ts

```diff
@@ -24,7 +24,7 @@ import NxThreatBarPage from './components/NxThreatBar/NxThreatBarPage';
 import NxRadioPage from './components/NxRadio/NxRadioPage';
 import NxVulnerabilityDetailsPage from './components/NxVulnerabilityDetails/NxVulnerabilityDetailsPage';
 import NxListPage from './styles/NxList/NxListPage';
-import NxTablePage from './styles/NxTable/NxTablePage';
+import NxTableStylePage from './styles/NxTable/NxTableStylePage';
```

Comment:
Don't forget to rename the `NxTablePage` within the `NxTableStylePage` file

Reply:
done

Reply:
> done

Doesn't seem to be

Reply:
bfc30d6